packages/graphback-datasync/src/deltaHelper.ts

```diff
@@ -0,0 +1,77 @@
+import { ModelDefinition, getTableName } from '@graphback/core';
+import { Db, Collection } from "mongodb";
+import { DataSyncFieldNames } from './util';
+
+export function getDeltaTableName(tableName: string) {
+  return `${tableName}Delta`
+}
+
+/**
+ * M
+ */
+export class MongoDeltaHelper {
+  protected collection: Collection;
+  public constructor(model: ModelDefinition, db: Db) {
+    this.collection = db.collection(getDeltaTableName(getTableName(model.graphqlType)));
+  }
+
+  public async insertDiff(updateDocument: any) {
+    const { _id,  ...delta } = updateDocument;
+    const version = delta[DataSyncFieldNames.version];
+    // eslint-disable-next-line @typescript-eslint/tslint/config
+    delete delta[DataSyncFieldNames.version];delete delta[DataSyncFieldNames.lastUpdatedAt];delete delta[DataSyncFieldNames.deleted]; 
```

Comment:
Why deleting those fields... and why in single line :)

Reply:
Deleting so as to ensure only the fields changed by the user are kept in the diff